specs/PermissionManagement.md

```diff
@@ -0,0 +1,554 @@
+Permission Management APIs
+===
+
+# Background
+
+The WebView2 team is adding support for more permission management scenarios. With the
+extended `PermissionRequested` event args and new permission APIs you will be able to:
+- Set permission for a site at any time
+- List all nondefault permissions
+- Control more permission kinds
+
+The new `ShouldPersist` property on the `PermissionRequestedEventArgs` lets you
+turn off the caching of permission state so that you can intercept all permission
+requests. By default, state set from the `PermissionRequested` event handler and default
+permission UI is cached across sessions and you may stop receiving PermissionRequested
+events for some permission kinds.
+
+The new APIs, `SetPermission` and `GetNonDefaultPermissionCollection`, provide
+all the information necessary to build a permission management page where a
+user can view and modify existing site permissions. The new permission kinds we will
+support are: local font list, automatic downloads, media autoplay, and file editing.
+
+See already supported [permission kinds](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1418.22#corewebview2_permission_kind)
+and existing [event args](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2permissionrequestedeventargs?view=webview2-dotnet-1.0.1418.22).
+
+
+# Examples
+
+## Extended PermissionRequestedEventArgs: ShouldPersist
```

Comment:
Should have subheads "C# sample" and "C++ sample" because the markdown does not separate the two code blocks clearly.